hydromt_wflow/utils.py

```diff
@@ -672,36 +794,36 @@ def convert_to_wflow_v1_sbm(
         "vertical.actevap": "land_surface__evapotranspiration_volume_flux",
         "vertical.actinfilt": "soil_water__infiltration_volume_flux",
         "vertical.excesswatersoil": "soil~compacted_surface_water__excess_volume_flux",
-        "vertical.excesswaterpath": "soil~non-compacted_surface_water__excess_volume_flux",  # noqa: E501
-        "vertical.exfiltustore": "soil_surface_water_unsat-zone__exfiltration_volume_flux",  # noqa: E501
+        "vertical.excesswaterpath": "soil~non-compacted_surface_water__excess_volume_flux",  # noqa : E501
+        "vertical.exfiltustore": "soil_surface_water_unsat-zone__exfiltration_volume_flux",  # noqa : E501
         "vertical.exfiltsatwater": "land.soil.variables.exfiltsatwater",
         "vertical.recharge": "soil_water_sat-zone_top__net_recharge_volume_flux",
         "vertical.vwc_percroot": "soil_water_root-zone__volume_percentage",
         "lateral.land.q_av": "land_surface_water__volume_flow_rate",
         "lateral.land.h_av": "land_surface_water__depth",
         "lateral.land.to_river": "land_surface_water~to-river__volume_flow_rate",
         "lateral.subsurface.to_river": "subsurface_water~to-river__volume_flow_rate",
+        "lateral.subsurface.drain.flux": "land_drain_water~to-subsurface__volume_flow_rate",  # noqa : E501
+        "lateral.subsurface.flow.aquifer.head": "subsurface_water__hydraulic_head",
+        "lateral.subsurface.river.flux": "river_water~to-subsurface__volume_flow_rate",
+        "lateral.subsurface.recharge.rate": "subsurface_water_sat-zone_top__net_recharge_volume_flow_rate",  # noqa : E501
         "lateral.river.q_av": "river_water__volume_flow_rate",
         "lateral.river.h_av": "river_water__depth",
         "lateral.river.volume": "river_water__instantaneous_volume",
         "lateral.river.inwater": "river_water_inflow~lateral__volume_flow_rate",
         "lateral.river.floodplain.volume": "floodplain_water__instantaneous_volume",
         "lateral.river.reservoir.volume": "reservoir_water__instantaneous_volume",
-        "lateral.river.reservoir.totaloutflow": "reservoir_water~outgoing__volume_flow_rate",  # noqa: E501
+        "lateral.river.reservoir.totaloutflow": "reservoir_water~outgoing__volume_flow_rate",  # noqa : E501
         "lateral.river.lake.storage": "lake_water__instantaneous_volume",
         "lateral.river.lake.totaloutflow": "lake_water~outgoing__volume_flow_rate",
     }
 
-    # Options in model section that were renamed
-    model_options = {
-        "masswasting": "gravitational_snow_transport",
-    }
-
     # Options in input section that were renamed
     input_options = {
-        "ldd": "local_drain_direction",
+        "ldd": "basin__local_drain_direction",
         "river_location": "river_location__mask",
-        "subcatchment": "subcatchment_location__count",
+        "subcatchment": "subbasin_location__count",
+        "altitude": "land_surface__elevation",
```

Comment:
Should be removed: input.static.land_surface__elevation and not input.land_surface__elevation